packages/governance/src/contractGovernance/paramManager.js

```diff
@@ -273,6 +281,33 @@ const makeParamManagerBuilder = zoe => {
     return harden(descriptions);
   };
 
+  /** @type {UpdateParams} */
+  const updateParams = async paramChanges => {
+    const asynchResults = [];
+    for (const changeSpec of paramChanges) {
```

Comment:
to avoid mutating,
```
const asyncResults = paramChanges.map(s => setters[`prepareToUpdate${s.parameterName}`](s.proposedValue)));
```
or if that's too dense,
```
const asyncResults = paramChanges.map(s => {
  const prepare = setters[`prepareToUpdate${s.parameterName}`];
  return prepare(s.proposedValue);
});
```

Reply:
Let me try this one again. I think it works with the refactoring I've done.

Reply:
done.